.github/workflows/velox_backend.yml

```diff
@@ -513,33 +515,28 @@ jobs:
         run: |
           yum update -y && yum install -y java-1.8.0-openjdk-devel wget git
           $SETUP install_maven
-      - name: Build for Uniffle 0.9.0
+      - name: Install Uniffle
```

Comment:
It is better to use `${{ matrix.uniffle }}` after uniffle, so that we can directly recorgnize the uniffle version used in workflow.

Reply:
thank you, addressed